book-a-reading-room-visit.web/wwwroot/css/sass/_booking-system.scss

```diff
@@ -636,8 +636,9 @@ form {
 
 #room-selection-form {
     fieldset {
-        border: 2px solid #ddd !important;
-        border-bottom: 4px solid #bbb !important;
+        background: rgba(19, 69, 113, 0.12);
+        border: 1px solid #134571 !important;
```

Comment:
Hi @JamesWChan - I think it would be better to achieve this without the use of `!important` by increasing the specificity slightly it should be possible to get the same result using the cascade.